src/parsing/parsers/generic/jison/sql_main.jison

```diff
@@ -65,24 +65,31 @@ SqlStatements_EDIT
  ;
 
 SqlStatement
- : DataDefinition
- | DataManipulation
- | QuerySpecification
+ : SqlStatementTypes
  ;
 
 SqlStatement_EDIT
  : AnyCursor
    {
-     parser.suggestDdlAndDmlKeywords();
+     parser.suggestDdlAndDmlKeywords(['EXPLAIN']);
    }
  | CommonTableExpression 'CURSOR'
    {
      parser.suggestKeywords(['SELECT']);
    }
- | DataDefinition_EDIT
+ | SqlStatementTypes_EDIT
+ ;
+
+SqlStatementTypes
+ : DataDefinition
+ | DataManipulation
+ | QuerySpecification
+ ;
```

Comment:
I like that it becomes more easy to extend this, but the naming is bad, and I don't know how to fix it. It's not fixable. It's name should be `SqlStatementWithoutExplain`, and there's no way around it.

I suggest to rollback this change and copy-paste all the types in the explain grammar

Reply:
I'd do it later. More parser experience will definitely come in handy and help us to name it properly!